src/tests/test_config.py

```diff
@@ -0,0 +1,13 @@
+import unittest
+from settings import load_configuration
+
+
+class TestSettings(unittest.TestCase):
+    def test_load_configuration(self):
+        """
+        Testing load configuration function
+        """
+        config = load_configuration()
+        self.assertEqual(config['debug'], True)
+        self.assertEqual(config['host'], '0.0.0.0')
+        self.assertEqual(config['port'], 8083)
```

Comment:
Notes that these settings can be obtained through environment variables, depending on how the deployment is done in the production environment, these variables can change and then these tests will break. I prefer that don't have these tests.